### DISCUSSION TITLE
請問石門山步道附近場地的概略位置?

### DISCUSSION THREAD

**Discussion by bestian** [bestian/playback#13] [Q&A] [answered]
@leechiuhui 謝謝提供場地說明和照片

想請您有空進一步提供該場地可公開的地址或概略位置

石門山步道附近的一個Google Map圖釘或座標連結

在此留言串回覆即可，我會再整理上Wiki~謝謝~

**Comment by leechiuhui** [comment, marked as answer]
桃園市龍潭區民有路附近
https://maps.app.goo.gl/ao92aKUd1DWa74p47

**Comment by bestian** [comment]
謝謝，已key in

--Bestian(宗浩)

=====

Be的網路工坊 <https://hack.bestian.tw> x 從心出發Playback
<https://playback.bestian.tw>

=====




Drago ***@***.***> 於 2024年10月7日 週一 下午4:13寫道：
